models/intel/yolo-v2-ava-0001/description/yolo-v2-ava-0001.md

```diff
@@ -2,7 +2,7 @@
 
 ## Use Case and High-Level Description
 
-This is a re-implemented and re-trained version of [YOLO v2](https://arxiv.org/abs/1612.08242) object detection network trained with VOC2012 training dataset.
+This is a reimplemented and retrained version of the [YOLO v2](https://arxiv.org/abs/1612.08242) object-detection network trained with the VOC2012 training dataset.
```

Comment:
I'm not sure it's correct to add a hyphen there. It's a network for object detection, so an object detection network. Even the abstract of the linked article says "object detection system".